ml-agents/mlagents/trainers/settings.py

```diff
@@ -297,6 +303,12 @@ def apply(self, key: str, env_channel: EnvironmentParametersChannel) -> None:
 class ConstantSettings(ParameterRandomizationSettings):
     value: float = 0.0
 
+    def __str__(self) -> str:
+        """
+        Helper method to output sampler stats to console.
+        """
+        return f"Float: value={self.value}"
```

Comment:
Another approach would be to use `__repr__`, and make the string sufficient to reconstruct the object:

>[ If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment).](https://docs.python.org/3/reference/datamodel.html#object.__repr__)

So in this case it would be something like `return f"ConstantSettings(value={self.value})`